main.go

```diff
@@ -95,13 +110,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	cfg, err := sharedConfig.LoadFrom(configFile)
+	cfg, err := sharedConfig.LoadFrom(args.ConfigFile)
 	if err != nil {
 		setupLog.Error(err, "Failed to load the configuration")
 		os.Exit(1)
 	}
 
-	strg, err := tokenstorage.NewVaultStorage("spi-controller-manager", cfg.VaultHost, cfg.ServiceAccountTokenFilePath, devmode)
+	strg, err := tokenstorage.NewVaultStorage("spi-controller-manager", cfg.VaultHost, cfg.ServiceAccountTokenFilePath, false)
```

Comment:
Having the ability to configure insecure connection to vault comes in handy when running the operator locally and vault in in minikube for example.